src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimCallbackHandlerTest.java

```diff
@@ -102,6 +107,11 @@
 @ExtendWith(SpringExtension.class)
 @SpringBootTest(classes = {
```

Comment:
Have we not agreed to remove spring context tests from unit tests?

Reply:
I believe are Integration tests